airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/UsagePerConnectionTable.tsx

```diff
@@ -139,31 +145,45 @@ const UsagePerConnectionTable: React.FC<UsagePerConnectionTableProps> = ({ credi
             <FormattedMessage id="credits.usage" />
           </SortableTableHeader>
         ),
-        accessor: "creditsConsumed",
-        collapse: true,
-        customPadding: { right: 0 },
-        Cell: ({ cell }: CellProps<FullTableProps>) => <UsageValue>{cell.value}</UsageValue>,
+        accessorKey: "creditsConsumed",
+        meta: {
+          collapse: true,
+          customPadding: {
+            right: 0,
+          },
+        },
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        cell: (props: CellContext<FullTableProps, string>) => <UsageValue>{props.cell.getValue()}</UsageValue>,
       },
       {
-        Header: "",
-        accessor: "creditsConsumedPercent",
-        customPadding: { left: 0 },
-        Cell: ({ cell }: CellProps<FullTableProps>) => <UsageCell percent={cell.value} />,
+        header: "",
+        accessorKey: "creditsConsumedPercent",
+        meta: {
+          customPadding: {
+            left: 0,
+          },
+        },
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        cell: (props: CellContext<FullTableProps, number>) => <UsageCell percent={props.cell.getValue()} />,
       },
       // TODO: Replace to Grow column
       {
-        Header: "",
-        accessor: "connectionId",
-        Cell: <div />,
-        customWidth: 20,
+        header: "",
+        accessorKey: "connectionId",
+        cell: () => <div />,
+        meta: {
+          customWidth: 20,
+        },
       },
     ],
     [onSortClick, sortBy, sortOrder]
   );
 
   return (
     <Content>
-      <Table columns={columns} data={sortingData} light />
+      <NextTable<FullTableProps> columns={columns} data={sortingData} light />
```

Comment:
I did some testing to remove the @ts-ignore but ended up with an odd error here:
![Screenshot 2023-01-09 at 1 54 45 PM](https://user-images.githubusercontent.com/4967694/211385750-d60e33d3-5a6d-4866-8410-51adefcbcd44.png)
Seems like something can be improved in the type chain.

Reply:
[Here](https://github.com/airbytehq/airbyte/blob/2713fa86192a20ed0c3964be47fbfd8669fda331/airbyte-webapp/src/components/ui/NextTable/NextTable.tsx#L14)
`columns: Array<ColumnDef<TData<T>, string>>;` seems to fix it.